nautobot/docs/release-notes/version-2.2.md

```diff
@@ -66,6 +66,53 @@ The CSV import functionality for all models has been changed from a synchronous
 Model CharFields' `max_length` attributes have been standardized globally to have at least 255 characters except where a shorter `max_length` is explicitly justified.
 
 <!-- towncrier release notes start -->
+## v2.2.1 (2024-04-15)
```

Comment:
As a housekeeping item, we should also make sure that `develop` includes the v1.6.18 release notes.